documentation/docs/11-cli.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Command Line Interface
+---
+
+SvelteKit includes a command line interface for building and running your app.
+
+In the default project template `svelte-kit dev`, `svelte-kit build` and `svelte-kit start` are aliased as `npm run dev`, `npm run build` and `npm start` respectively. You can also invoke the CLI with [npx](https://www.npmjs.com/package/npx) (or your package manager's equivalent, if you're not using npm):
```

Comment:
I'm pretty sure npx still works even if you installed deps with pnpm or Yarn, since those all write binaries to `node_modules/.bin`. Not sure whether this is worth mentioning.